packages/server/src/content-providers/local/fs.js

```diff
@@ -160,6 +160,18 @@ function get(
   });
 }
 
+function post(
+  options: DiskProviderOptions,
+  unsafeFilePath: string,
+  content: mixed
+) {
+  const filePath = path.join(
+    options.local.baseDirectory,
+    sanitizeFilePath(unsafeFilePath)
+  );
+  return fs.outputFile(filePath, JSON.stringify(content));
```

Comment:
Does this have a callback to indicate any errors?

Reply:
This returns a promise which I hooked up to [here](https://github.com/nteract/commuter/pull/242/files#diff-f8491e2a0aa61b0e31db01f53d7114b0R49). Hopefully. Didn't actually test it with something that fails.

Reply:
Derp, I see. That's clear as day now.